### PR TITLE
Add switch selector

### PR DESCRIPTION
**Pull request by Cloud11PL**
Adds switch button selector as per https://app.clickup.com/t/2549495/SALEOR-6155 according to design https://www.figma.com/file/JnN0HR5ZNYH8SN2cSbKkvK/MacawUI?node-id=5%3A2256

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6155 Change dialog of the shipping edit](https://app.clickup.com/t/2549495/SALEOR-6155)

File: src/SwitchSelector/SwitchSelector.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Meta, Story } from "@storybook/react";
```
Comment (dominik-zeglen) [inline src/SwitchSelector/SwitchSelector.stories.tsx:1]: Can we merge these stories into one, use card decorator and write a sentence or two of description?
We dont maintain any technical documentation, so at least keep this one pretty 🙃

File: src/SwitchSelector/SwitchSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}));
+
+export const SwitchSelector: React.FC<SwitchSelectorProps> = ({
+  options,
+  setActiveTab,
+  activeTab,
+}) => {
+  const classes = useStyles();
+
+  return (
```
Comment (dominik-zeglen) [inline src/SwitchSelector/SwitchSelector.tsx:51]: If it supports more than two options, then i think its better to go for compound component pattern. Mapping children led us in dashboard to constant need for extending choice interface.

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

File: src/SwitchSelector/SwitchSelector.tsx (hunk 2 above)
Comment (Cloud11PL) [reply src/SwitchSelector/SwitchSelector.tsx:51]: So I should make two components, one that is a wrapper eg. `SwitchSelector` and the buttons eg. `SwitchSelectorButton` in macaw, then in dashboard do something like:

```
<SwitchSelector>
   <SwitchSelectorButton name={optionOneName} onClick={onClick}>{optionOneLabel}</SwitchSelectorButton>
   <SwitchSelectorButton name={optionTwoName} onClick={onClick}>{optionTwoLabel}</SwitchSelectorButton>
   <SwitchSelectorButton name={optionThreeName} onClick={onClick}>{optionThreeLabel}</SwitchSelectorButton>
   ...
</SwitchSelector>
```
?

**Review by dominik-zeglen — APPROVED** [review]

**Review by jwm0 — CHANGES_REQUESTED** [review]
See Chromatic discussion

**Comment by Cloud11PL** [comment]
@jwm0 fixed

**Review by jwm0 — APPROVED** [review]

**Pull request merged**